<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>项目(&amp;P)</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionDeleteProject"/>
    <addaction name="actionSettingProject"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCase">
    <property name="title">
     <string>用例(&amp;U)</string>
    </property>
    <addaction name="actionNewUseCase"/>
    <addaction name="actionDeleteUseCase"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="actionNFCProject"/>
    <addaction name="actionCENIProject"/>
    <addaction name="actionSwitchProject"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuCase"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewProject">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionDeleteProject">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionSettingProject">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionNewUseCase">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionDeleteUseCase">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionNFCProject">
   <property name="text">
    <string>NFC项目</string>
   </property>
  </action>
  <action name="actionCENIProject">
   <property name="text">
    <string>CENI项目</string>
   </property>
  </action>
  <action name="actionSwitchProject">
   <property name="text">
    <string>交换机项目</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
